composer.json

```diff
@@ -12,13 +12,14 @@
         }
     },
     "require": {
-        "php": "^7.1",
+        "php": "^8.1",
```

Comment:
If you still need 7.x support, I think you can do something like this
```suggestion
        "php": "^7.1|^8.1",
```

Reply:
We certainly hope not to need PHP 7.x support 🤣

Reply:
Yeah, I think if anybody needs 7.X support they can just use an old release at this point.